integration/kubernetes/init.sh

```diff
@@ -51,6 +51,9 @@ kubeadm_config_file="$(mktemp --tmpdir kubeadm_config.XXXXXX.yaml)"
 sed -e "s|CRI_RUNTIME_SOCKET|${cri_runtime_socket}|" "${kubeadm_config_template}" > "${kubeadm_config_file}"
 sed -i "s|KUBERNETES_VERSION|v${kubernetes_version/-*}|" "${kubeadm_config_file}"
 
+if [ "${BAREMETAL}" == true ] && [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
```

Comment:
:-) fun test. You probably could have gotten away with `sudo swapoff -a || true` :-)
Yeah, I hit this swap issue every time I reboot one of my local test machines ;-)

Reply:
it looks like that this error only occur in bare mental???? I'm wondering that's because VM won't get any swap allocation???